### ISSUE TITLE
react-hot-loader is being deprecated, replace with react-transform-hmr

### ISSUE THREAD

**Issue by jasongonzales23** [shakacode/react-webpack-rails-tutorial#101]
I will be happy to do this and submit a PR. Just logging the issue.

**Comment by alexfedoseev** [comment]
@jasongonzales23 Thanks! But let's start new PR after #96 will be merged. There will be a lot of changes in `client` dir, so it's better to start working with updated configs and structure.

**Comment by justin808** [comment]
@alexfedoseev Let's get this in there ASAP, so we can start the next few PRs.

**Comment by alexfedoseev** [comment]
@jasongonzales23 #96 is merged. Feel free to submit a PR. Thanks!

**Comment by jasongonzales23** [comment]
OK, I will get on it!

**Comment by alexfedoseev** [comment]
@jasongonzales23 Replaced in https://github.com/shakacode/react-webpack-rails-tutorial/pull/109

**Comment by jasongonzales23** [comment]
Ah thanks for beating me to it. Been swamped lately!

**Comment by alexfedoseev** [comment]
NP :) Closing this.
